zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -293,7 +295,7 @@ class Handlers(queryExtractor: QueryExtractor) {
         "width" -> (if (width < 0.1) 0.1 else width),
         "depth" -> (depth + 1) * 5,
         "depthClass" -> (depth - 1) % 6,
-        "children" -> spanMap.get(span.id).map(s => SpanId(s.id).toString).mkString(","),
+        "children" -> byParentId.getOrElse(span.id, Nil).map(s => SpanId(s.id).toString).mkString(","),
```

Comment:
This is basicalle #1014, which was needed here, since it's implemented that way in JS.